Mage.Tests/src/test/java/org/mage/test/cards/triggers/IndominusRexAlphaTests.java

```diff
@@ -0,0 +1,62 @@
+package org.mage.test.cards.triggers;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+/**
+ *
+ * @author jimga150
+ */
+public class IndominusRexAlphaTests extends CardTestPlayerBase {
+
+    @Test
+    public void IndominusRexAlphaTest() {
```

Comment:
style - method names should start with lowercase character (e.g. testIndominusRexAlpha)